src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -461,6 +470,16 @@ internal void LoadResponse(FunctionLoadResponse loadResponse)
             _inputLinks.Add(disposableLink);
         }
 
+        internal void LoadResponse(FunctionLoadResponseCollection loadResponseCollection)
```

Comment:
Just curious, how often does FunctionLoadResponse / FunctionLoadResponseCollection get called? Is it just on startup/restart? Or more frequently than that?

Reply:
It will be called frequently than that. Host sends request to worker to load functions and gets load responses. So, 1 call for each function in the app. They can reduce number of calls by sending/receiving collection instead of one by one.